### PR TITLE
Add tests to `report.rs`

### PR DESCRIPTION
**Pull request by hkctkuy**

### CODE REVIEW

File: libcasr/src/report.rs [hunk 1]
```diff
@@ -774,3 +774,230 @@ pub fn cluster_reports(casreps: &[CrashReport]) -> Result<Vec<u32>> {
...
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::constants::*;
+    use crate::init_ignored_frames;
+
+    #[test]
+    fn test_report_display() {
+        // TODO: Fill all empty fields
+        let report = CrashReport {
```
Comment (anfedotoff) [inline libcasr/src/report.rs:787]: You can call  `CrashReport::new()` [function](https://github.com/ispras/casr/blob/master/libcasr/src/report.rs#L239) first.

File: libcasr/src/report.rs [hunk 2]
```diff
@@ -774,3 +774,230 @@ pub fn cluster_reports(casreps: &[CrashReport]) -> Result<Vec<u32>> {
...
+            architecture: "amd64".to_string(),
+            executable_path: "/home/hkctkuy/github/casr/casr/tests/casr_tests/bin/load_fuzzer".to_string(),
+            proc_environ: vec!["CARGO=/home/hkctkuy/.rustup/toolchains/nightly-x86_64-unknown-linux-gnu/bin/cargo".to_string()],
+            proc_cmdline: "/home/hkctkuy/github/casr/casr/tests/casr_tests/bin/load_fuzzer /home/hkctkuy/github/casr/casr/tests/casr_tests/casrep/libfuzzer_crashes_xlnt/crash-49c4e6270849ab5c9c13f64b98f817efa840d8a3".to_string(),
+            stdin: "/home/hkctkuy/github/casr/casr/tests/casr_tests/casrep/afl-out-xlnt-small/afl_main-worker/crashes/id:000007,sig:00,sync:afl_s01-worker,src:000371".to_string(),
+            proc_status: vec!["process 2945397".to_string()],
+            proc_maps: vec![
+                "      0x555555554000     0x555555556000     0x2000        0x0 /usr/local/bin/tiff2pdf".to_string(),
+            ],
+            stacktrace: vec![
```
Comment (anfedotoff) [inline libcasr/src/report.rs:809]: You could decrease the size of vec![] of up to two elements, for all `*_report` fields too.

File: libcasr/src/report.rs [hunk 3]
```diff
@@ -774,3 +774,230 @@ pub fn cluster_reports(casreps: &[CrashReport]) -> Result<Vec<u32>> {
...
+                severity: "NOT_EXPLOITABLE".to_string(),
+                short_description: "SourceAv".to_string(),
+                description: "Access violation on source operand".to_string(),
+                explanation: "The target crashed on an access violation at an address matching the source operand of the current instruction. This likely indicates a read access violation.".to_string(),
+            },
+            pid: 16476,
+        };
+        assert_eq!(
+            159,
+            format!("{report}").split('\n').collect::<Vec<&str>>().len()
```
Comment (anfedotoff) [inline libcasr/src/report.rs:952]: We need to check not only size, but content too (maybe for some critical parts only).

File: libcasr/src/report.rs (hunk 1 above)
Comment (hkctkuy) [reply libcasr/src/report.rs]: Done

File: libcasr/src/report.rs (hunk 2 above)
Comment (hkctkuy) [reply libcasr/src/report.rs]: Done

File: libcasr/src/report.rs (hunk 3 above)
Comment (hkctkuy) [reply libcasr/src/report.rs]: Done

**Review by anfedotoff — APPROVED** [review]

**Review by Avgor46 — APPROVED** [review]

**Pull request merged**